builders/pallets-precompiles/precompiles/proxy.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Interacting with the Proxy Precompile on Moonbeam
+description: How to use the Moonbeam proxy Solidity precompile interface to make substrate calls on an account's behalf
+keywords: solidity, ethereum, proxy, moonbeam, precompiled, contracts, substrate
+---
+
+# Interacting with the Proxy Precompile
+
+![Proxy Moonbeam Banner](/images/builders/pallets-precompiles/precompiles/proxy/proxy-banner.png)
+
+## Introduction {: #introduction } 
+
+The proxy precompile on Moonbeam allows accounts to set proxy accounts that can perform specific limited actions on their behalf, such as governance, staking, or balance transfers.
+
+If a user wanted to provide a second user access to a limited number of actions on their behalf, traditionally the only method to do so would be by providing the first account's private key to the second. However, Moonbeam has included the [Substrate proxy pallet](/builders/pallets-precompiles/pallets/proxy){target=_blank}, which enables proxy accounts. Proxy accounts ought to be used due to the additional layer of security that they provide, where many accounts can do actions for a main account. This is best if for example a user wanted to keep their wallet safe in cold storage but still wanted to access parts of the wallet's functionality like governance or staking.  
+
+To learn more about proxy accounts and how to set them up for your own purposes without use of the proxy precompile, view the [Setting up a Proxy Account](/tokens/manage/proxy-accounts){target=_blank} page.
+
+The proxy precompile is located at the following address:
+
+=== "Moonbase Alpha"
+     ```
+     {{networks.moonbase.precompiles.proxy}}
+     ```
+
+## The Proxy Solidity Interface {: #the-proxy-solidity-interface } 
+
+[`Proxy.sol`](https://github.com/PureStake/moonbeam/blob/master/precompiles/proxy/Proxy.sol){target=_blank} is an interface through which Solidity contracts can interact with the proxy pallet. You will not have to be familiar with the Substrate API, since you can interact with it using the Ethereum interface you're familiar with.
```

Comment:
```suggestion
[`Proxy.sol`](https://github.com/PureStake/moonbeam/blob/master/precompiles/proxy/Proxy.sol){target=_blank} is an interface through which Solidity contracts can interact with the proxy pallet. You do not have to be familiar with the Substrate API since you can interact with it using the Ethereum interface you're familiar with.
```